### PR TITLE
cx_Logging 3.0

### PR DESCRIPTION
**Pull request by marcelotduarte**
I need this package to release the new cx_Freeze 6.6

### CODE REVIEW

**Comment by Biswa96** [comment]
As this is a new package, I don't think the `python3` name conflict is required. But wait for python gurus.

File: mingw-w64-python-cx_Logging/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Maintainer: Marcelo Duarte https://github.com/marcelotduarte
+
+_realname=cx_Logging
+pkgbase=mingw-w64-python-${_realname}
```
Comment (lazka) [inline mingw-w64-python-cx_Logging/PKGBUILD:4]: please rename to 'mingw-w64-python-cx-logging' to be in line with the Arch naming (yes, I know cx_freeze is different right now)

File: mingw-w64-python-cx_Logging/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+pkgname=("${MINGW_PACKAGE_PREFIX}-python-${_realname}")
+provides=("${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
+conflicts=("${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
+replaces=("${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
```
Comment (lazka) [inline mingw-w64-python-cx_Logging/PKGBUILD:8]: all this python3 stuff can be removed

Comment (marcelotduarte) [reply mingw-w64-python-cx_Logging/PKGBUILD]: done

File: mingw-w64-python-cx_Logging/PKGBUILD (hunk 1 above)
Comment (marcelotduarte) [reply mingw-w64-python-cx_Logging/PKGBUILD]: wait a moment.

Comment (marcelotduarte) [reply mingw-w64-python-cx_Logging/PKGBUILD]: done

**Pull request merged**